src/components/base/Markdown.js

```diff
@@ -3,14 +3,15 @@ import PropTypes from 'prop-types';
 
 import Remarkable from 'react-remarkable';
 
-const Markdown = ({ src }) => (
-  <div className="Markdown">
+const Markdown = ({ src, fontSize = '' }) => (
+  <div className={`Markdown Markdown--${fontSize}`}>
```

Comment:
let's use a cx conditional so we don't end up with `Markdown--` when `fontSize` defaults to `''`